assets/sass/_content-post.scss

```diff
@@ -17,6 +17,17 @@
     text-transform: none;
   }
 
+  a.external::after {
```

Comment:
Oooo, nevermind, just doing two `:not`s.  This was cool.  Thanks for the tip and for the learning experience!